### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by natakazakova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,99 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/856911405/24.2.1%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1253420)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 [![](https://img.shields.io/badge/💬_Leave_Feedback-feecdd?style=flat-square)](#does-this-example-address-your-development-requirementsobjectives)
 <!-- default badges end -->
 # Reporting for ASP.NET Core - Integrate AI-powered Summarize and Translate Features based on Azure OpenAI
 
-Description
+This example adds two new buttons that help users to summarize and translate a report content in the JavaScript Document Viewer with AI-powered functionality:
```
Comment (Abadzhev) [inline README.md:8]:
```suggestion
This example extends the DevExpress JavaScript Report Viewer with AI-based capabilities. These capabilities are available in UI via two buttons that access the report document and process its content as follows:
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,99 @@
...
 
+- **Summarize**: When you click this button, Generative AI summaries generate content that helps you quickly understand/analyze core insights associated with a given report.
```
Comment (Abadzhev) [inline README.md:10]:
```suggestion
- **Summarize**: Uses generative AI to summarize report content and help you quickly review core insights associated with this report.
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,99 @@
...
+- **Translate**: This button helps you translate report documents, chosen pages, or selected content to a desired language with AI-powered translation tools.
```
Comment (Abadzhev) [inline README.md:11]:
```suggestion
- **Translate**: Uses AI services to translate report content to another language.
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,99 @@
...
+
+![AI-Powered Summarize and Translate Buttons](web-reporting-ai-enhancements.png)
```
Comment (Abadzhev) [inline README.md:13]:
```suggestion
The following image shows the application's interface. As you can see, users can process the entire document, individual pages, or selected content. 

![AI-Powered Summarize and Translate Buttons](web-reporting-ai-enhancements.png)
```

File: README.md [hunk 5]
```diff
@@ -1,13 +1,99 @@
...
+
+## Implementation Details
+
+### Add NuGet Packages
+
+Add the following NuGet packages:
+- `DevExpress.AIIntegration.AspNetCore.Reporting`
+- `DevExpress.AIIntegration.Azure.OpenAI` or `DevExpress.AIIntegration.OpenAI` based on your AI service preferences.
```
Comment (Abadzhev) [inline README.md:21]:
```suggestion
- `DevExpress.AIIntegration.Azure.OpenAI` or `DevExpress.AIIntegration.OpenAI` based on your AI service preferences. This project uses Azure OpenAI. The remainder of this document describes steps related to this package.  
```

File: README.md [hunk 6]
```diff
@@ -1,13 +1,99 @@
...
+
+### Add Personal Keys
+
+You need to create an Azure OpenAI resource in the Azure portal to use AI Assistants for DevExpress Reporting. Refer to the following help topic for details: [Microsoft - Create and deploy an Azure OpenAI Service resource](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
```
Comment (Abadzhev) [inline README.md:25]:
```suggestion
To use AI Assistants in your applications, you need to create an Azure OpenAI resource in the Azure portal . Refer to the following help topic for details: [Microsoft - Create and deploy an Azure OpenAI Service resource](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/create-resource?pivots=web-portal).
```

File: README.md [hunk 7]
```diff
@@ -1,13 +1,99 @@
...
+
+Once you obtain a private endpoint and an API key, open [appsettings.json](./CS/BlazorReportViewer/appsettings.json) and add `DeploymentName`, `AzureOpenAIKey`, and `AzureOpenAIEndpoint` values to the fields below (`DeploymentName` is a name of your Azure model, for example, `GPT4o`):
```
Comment (Abadzhev) [inline README.md:27]: I also changed the link to the JSON file. Seemed like copy/paste from the Blazor version. 

```suggestion
Once you obtain a private endpoint and an API key, open [appsettings.json](./CS/JSDocumentViewer/appsettings.json) and specify `DeploymentName`, `AzureOpenAIKey`, and `AzureOpenAIEndpoint` values. Note that `DeploymentName` is  set to `GPT4o`, but you can specify a different model:
```

File: README.md [hunk 8]
```diff
@@ -1,13 +1,99 @@
...
+    config.AddWebReportingAIIntegration(cfg =>
+        cfg.SummarizeBehavior = SummarizeBehavior.Abstractive);
+});
+
+var app = builder.Build();
+```
+
+### Register AI Services on the Client
+
+Open `DocumentViewer.cshtml` and create the JavaScript function that enables AI services and add languages. Call this function on the Document Viewer's [OnInitializing](https://docs.devexpress.devx/XtraReports/DevExpress.AspNetCore.Reporting.WebDocumentViewer.WebDocumentViewerClientSideEventsBuilderBase-2.OnInitializing(System.String)) event:
```
Comment (Abadzhev) [inline README.md:75]:
```suggestion
Open `DocumentViewer.cshtml` and create a JavaScript function that enables AI services and populates a language list. Call this function on the Document Viewer's [OnInitializing](https://docs.devexpress.devx/XtraReports/DevExpress.AspNetCore.Reporting.WebDocumentViewer.WebDocumentViewerClientSideEventsBuilderBase-2.OnInitializing(System.String)) event:
```

File: README.md (hunk 7 above)
Comment (natakazakova) [reply README.md:27]: Yes, I've missed this link. Thanks!

**Pull request merged**